src/components/TrottMap/MyMap.jsx

```diff
@@ -4,14 +4,44 @@ import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
 import { gql } from 'apollo-boost';
 import { useQuery } from '@apollo/react-hooks';
 import L from 'leaflet';
+import MarkerClusterGroup from 'react-leaflet-markercluster';
+import 'react-leaflet-markercluster/dist/styles.min.css';
 
-const myIcon = L.icon({
-  iconUrl: 'https://www.scafom-echafaudage.fr/wp-content/uploads/2018/10/marker_2-512.png',
-  iconSize: [60, 50],
+const tierIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/tier.png',
   iconAnchor: [12.5, 41],
   popupAnchor: [0, -41],
 });
 
+const dottIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/dott.png',
+  iconAnchor: [12.5, 41],
+  popupAnchor: [0, -41],
+});
+
+const birdIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/bird.png',
+  iconAnchor: [12.5, 41],
+  popupAnchor: [0, -41],
+});
+
+const voiIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/voi.png',
+  iconAnchor: [12.5, 41],
+  popupAnchor: [0, -41],
+});
+
+const windIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/wind.png',
+  iconAnchor: [12.5, 41],
+  popupAnchor: [0, -41],
+});
+
+const ponyIcon = L.icon({
+  iconUrl: 'https://cdn.fluctuo.com/markers/pony.png',
+  iconAnchor: [12.5, 41],
+  popupAnchor: [0, -41],
+});
 const posMarker = [48.866667, 2.333333];
 const defaultFluctuoAPI = gql`
 {
```

Comment:
is there a way to format the request like this:

    {
        provider {
          name
          vehicles (lat: ${posMarker[0]}, lng: ${posMarker[1]}) {
            id
            type
            attributes
            lat
            lng
          }
        }
    }

so you would have the vehicles directly by provider, so you can write later:

    data.map(provider => (
        <MarkerClusterGroup>
          {provider.vehicles.map(vehicle => <Marker></Marker>}
        </MarkerClusterGroup>
    ))